portfolio/src/main/java/com/google/sps/servlets/CommentServlet.java

```diff
@@ -75,7 +90,7 @@ private Comment entityToComment(Entity entity) {
     Date timestamp = (Date) entity.getProperty("timestamp");
     String message = (String) entity.getProperty("message");
     
-    Comment comment = new Comment(author, timestamp, message);
+    Comment comment = new Comment(author, userEmail, timestamp, message);
```

Comment:
maybe i'm misunderstanding here; are we loading a comment from the database but with the current user's email? Should it not be read from datastore?

Reply:
nope you're completely right 🤦‍♀️ thank you!